src/scss/pages.scss

```diff
@@ -160,124 +160,65 @@ app-user-billing {
 }
 
 
-/* Register Layout Page - Exempt from themify */
-body.theme_light_force {
-    background-color: #ECF0F5;
-    a, .btn-link {
-        color: #175DDC;
-        &:hover {
-            color: #104097;
-        }
-    }
-
-    .btn-outline-secondary {
-        color: #6c757d;
-        &:hover {
-            color: #212529;
+/* Register Layout Page */
+.layout {
+    &.enterprise2 {
+        header {
+            background: #175DDC;
+            color: #CED4DA;
+
+            &:before {
+                background: #175DDC;
+                content: "";
+                height: 416px;
+                left: 0;
+                position: absolute;
+                top: -76px;
+                transform: skewY(-3deg);
+                width: 100%;
+                z-index: -1;
+            }
+            img.logo {
+                margin: 12px 0 0;
+                max-width: 284px;
+                height: 57px;
```

Comment:
Can you alphabetise this to match the other elements?

Reply:
This is all done per the changes you sent me